nio/service/base.py

```diff
@@ -23,7 +23,7 @@ class BlockExecution(PropertyHolder):
     This information is parsed/used by the Router which is able then, based on
     the sending block, to forward signals to its receivers
     """
-    name = StringProperty(title="Name")
+    id = StringProperty(title="Id")
```

Comment:
No more service names!?!

Reply:
This 'id' refers refers to a block ('id') in the service execution, the service name is kept intact

Reply:
Oh yeah, my bad. Saw this was in the `service/base.py` and assumed it was the base service class. This makes sense